docs/sophia_syntax.md

```diff
@@ -255,9 +257,14 @@ In order of highest to lowest precedence.
 | `!` | right
 | `^` | left
 | `*` `/` `mod` | left
+| `bnot` | right
```

Comment:
I would put `bnot` at the same level as `!`. This is what both C and Erlang does.